run.py

```diff
@@ -13,15 +13,21 @@
 def matlab_command(gem):
     cmd = """
         cd {}geckomat
-        model = load('{}');
+        if endsWith('{}','.xml')
+            model = importModel('{}');
+        elseif  endsWith('{}','.mat')
+            model = load('{}');
+        else
+            disp('Model file format is not compatible with GECKO')
+        end
```

Comment:
Sweet!
How about using `error` instead of `disp`?

Reply:
Done!